src/components/DiagramFrame/SeqDiagram/LifeLineLayer/LifeLineLayer.vue

```diff
@@ -1,28 +1,41 @@
 <template>
   <div
-    class="life-line-layer lifeline-layer absolute h-full flex flex-col pt-8"
-    :style="{ 'min-width': '200px' }"
+    class="life-line-layer lifeline-layer absolute h-full flex flex-col pt-8 top-0"
```

Comment:
why top-0 here?

Reply:
There was already an `absolute` setted on it, but without a `top`, it would still be push down by the previours message-layer.